operations/patch_module.go

```diff
@@ -57,14 +57,23 @@ func PatchSetModule() cli.Command {
 				return errors.Wrap(err, "problem accessing evergreen service")
 			}
 
-			existingPatch, err := ac.GetPatch(patchID)
+			var existingPatch *patch.Patch
+			if patchID == "" {
+				patchList, err := ac.GetPatches(1)
+				if err != nil {
+					return errors.Wrapf(err, "problem getting patches from user")
+				}
+				existingPatch = &patchList[0]
```

Comment:
should we make sure that there is at least one patch in the list before accessing the index, so that we can avoid out of bounds errors?

Reply:
added check